ServerHttp/src/test/kotlin/io/raspberrywallet/server/MainVerticleTest.kt

```diff
@@ -0,0 +1,126 @@
+package io.raspberrywallet.server
+
+import io.raspberrywallet.mock.ManagerMock
+import io.raspberrywallet.server.Server.Companion.PORT
+import io.vertx.core.Vertx
+import io.vertx.ext.unit.TestOptions
+import io.vertx.ext.unit.TestSuite
+import io.vertx.ext.unit.report.ReportOptions
+import io.vertx.kotlin.core.json.get
+import org.junit.jupiter.api.Test
+
+
+internal class MainVerticleTest {
+    private val testOptions = TestOptions()
+        .addReporter(ReportOptions().setTo("console"))
+        .setTimeout(1000)
+    val host = "localhost"
+
+    @Test
+    fun test() {
```

Comment:
This test method is huge, could you split it to one method-one test?

Reply:
I'm not using JUnit here, and due to nature of Vertx Unit, it requires to write test in this (asynchronous) way. Docs: https://vertx.io/docs/vertx-unit/java/
Anyway I made some changes in next commit, but I'm not sure if that's better